provisioning/roles/mangaki_source/README.md

```diff
@@ -85,6 +85,9 @@ mangaki_source_auth_providers: ['twitter', 'facebook']
 # The path to the web root to store static files in (required).
 mangaki_source_static_root: '/var/www/mangaki/static'
 
+# The path to the web root to store data and snapshots in (required).
+mangaki_source_data_root: '/var/mangaki/data'
```

Comment:
Use `/opt/mangaki/data` for consistency with the default vars of standalone.

Reply:
I really don't get it. The other path is `/opt/mangaki`. I put `/opt/mangaki/data` everywhere.